ServerCore/Pages/Teams/Disqualify.cshtml

```diff
@@ -0,0 +1,130 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/Disqualify"
+@model ServerCore.Pages.Teams.DisqualifyModel
+@{
+    ViewData["Title"] = "Disqualify Team";
+    Layout = "_teamLayout.cshtml";
+}
+
+<h2>@Html.DisplayFor(model => model.Team.Name): Disqualify</h2>
+<div>
+    <a asp-page="./Details" asp-route-teamId="@Model.Team.ID">Cancel</a>
+</div>
+
+@if (@Model.Team.IsDisqualified)
+{
+    <h3>Are you sure you want to undo disqualification of this team?</h3>
+}
+else
+{
+    <h3>Are you sure you want to disqualify this team?</h3>
+}
+
+<div>
+    <hr />
+    <dl>
+        <!-- class="dl-horizontal"> -->
+        <dt>
+            Team name
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.Team.Name)
+        </dd>
+        <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt)
+    <dt>
```

Comment:
I'm not sure we need all of the details about the team here. Name and perhaps contact info seems like enough since we can use the details page for the rest